service/src/main/java/bio/terra/workspace/app/controller/ReferencedGcpResourceController.java

```diff
@@ -128,15 +127,6 @@ private AuthenticatedUserRequest getAuthenticatedInfo() {
     return new ResponseEntity<>(referenceResource.toApiResource(), HttpStatus.OK);
   }
 
```

Comment:
Why are there Java changes? Is this a clean merge?

Reply:
This is part of removing the deprecated APIs